package.json

```diff
@@ -21,35 +21,37 @@
   },
   "devDependencies": {
     "@babel/eslint-plugin": "7.14.5",
+    "@babel/eslint-parser": "7.15.0",
     "@jackfranklin/test-data-bot": "1.3.0",
     "concurrently": "5.3.0",
-    "eslint": "6.6.0",
+    "eslint": "7.11.0",
     "eslint-plugin-node": "11.1.0",
     "eslint-plugin-react": "7.24.0",
     "eslint-plugin-react-hooks": "4.2.0",
     "json-server": "0.16.3",
     "stylelint": "13.13.1",
     "stylelint-scss": "3.19.0",
-    "stylelint-webpack-plugin": "1.2.3"
+    "stylelint-webpack-plugin": "1.2.3",
+    "typescript": "4.4.2"
```

Comment:
Why add typescript? Just to give a candidate the option to use it?

Reply:
This is the one I felt most conflicted on. Technically it's a peer dependency of tsutils with is a dependency of a dependency of react-scripts. Part of me wants to just keep that peer dependency warning rather than add ts. Thoughts?

Reply:
I guess the question is what might cause a candidate more confusion, having typescript included when it's not used anywhere or having a peer dependency warning?

Reply:
The candidate has always had the option to use TS, so adding this dep doesn't really change that, although I guess it makes it a little more explicit.

I think it's good practice for us to clean up those peer deps, so I'm good to move ahead with this.